### PR TITLE
Localizing some code

### PR DESCRIPTION
**Pull request by clusterfack**
We really need to organize all this code better. Primarily moving _DEFINES to actually be before everything else.

### CODE REVIEW

File: code/_onclick/hud/parallax.dm [hunk 1]
```diff
@@ -6,6 +6,9 @@
 #define GRID_WIDTH 3
 
 var/list/parallax_on_clients = list()
+var/parallax_initialized = 0
+var/space_color = "#050505"
+var/list/parallax_icon[(GRID_WIDTH**2)*3]
```
Comment (ComicIronic) [inline code/_onclick/hud/parallax.dm:11]: Why 3?

Comment (clusterfack) [inline code/_onclick/hud/parallax.dm]: Because I said so

Comment (clusterfack) [inline code/_onclick/hud/parallax.dm]: As you see below three different loops for three different planes of parallax

Comment (ghost) [reply code/_onclick/hud/parallax.dm:11]: @clusterfack he wants it to be a define. I'll do the work for you
`#define THREE 3`
just copy that code in and replace the 3 with that define

**Pull request merged**